app/views/categories/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<div id="cat-index" class="container-fluid col-sm-12 col-md-6 offset-md-3 col-lg-8 offset-lg-2 h-100">
+  <%= render partial: 'shared/navbar' %>
+  <h1 class="fw-bold" style="text-align: center; font-family: proxima-nova, sans-serif;
+    margin-top: 15px; margin-bottom: 25px;">Categories Index</h1>
```

Comment:
- [ ] Great job so far! However, according to the design, the title should be in the navbar. I recommend that you reposition it and adjust the color.

![image](https://github.com/jcmunav63/rails-budget-app/assets/55840999/9ce32e9e-a3a2-4d68-ae70-756d53df3a42)